components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -483,7 +483,8 @@ def pickFirstSignature(condition, filterLambda):
             else:
                 # Just throw; we have no idea what we're supposed to
                 # do with this.
-                caseBody.append(CGGeneric("return Throw(cx, NS_ERROR_XPC_BAD_CONVERT_JS);"))
+                caseBody.append(CGGeneric("throw_internal_error(cx, \"NS_ERROR_XPC_BAD_CONVERT_JS\");\n"
```

Comment:
Let's pass "Could not convert JavaScript argument" instead: https://dxr.mozilla.org/mozilla-central/source/js/xpconnect/src/xpc.msg#20